package_builder/install_claw_release.py

```diff
@@ -94,6 +94,28 @@ def get_ant_dir(machine: str) -> Optional[str]:
         assert False, 'Unknown machine'
 
 
+def set_permissions(dir: str, symlink: str):
+
+    def set_read(path: str):
```

Comment:
could you add a comment saying this is setting permission for "all"
I also don't see below which one means "all"

Reply:
Function does not set permissions for all, only for others. "Others" in this case are everyone except Jenkins and members of its group. 
S_IROTH =  Read permission bit for other users
S_IXOTH = execute/search permission, others